### PR TITLE
Disabling nodes functionality

### PR DESCRIPTION
**Pull request by idrissou**

### CODE REVIEW

File: app/components/map/mapDirective.js [hunk 1]
```diff
@@ -437,6 +437,43 @@ app.directive('d3Map', ['$rootScope', 'toolbar', 'toolbarItems', function($rootS
...
+                    }
+                })
+            }
+
+            /** Delete  Disabled Map nodes
+             * @returns none
+             */
+            // This function should be called after DisableSelectedElements(), when disabled nodes are aimed to be removed
+
+            function DeleteDisabledNodes(){
```
Comment (jidel) [inline app/components/map/mapDirective.js]: Do we need to delete nodes?

**Comment by jidel** [comment]
References #73

**Comment by idrissou** [comment]
We need to, in the case we are creating a new map from an existing map.
You only keep the nodes you need for the new map.
This is a sub-category functionality, for the main functionality.

**Comment by jidel** [comment]
What if we just take the nodes we want, copy them (in the controller) and create a new map from that? no need to delete nodes, right?

**Pull request merged**